README.md

```diff
@@ -4,6 +4,62 @@
 
 Rust backend for napari contains code to speed up triangulation.
 
+## Usage
+
+Currently, this package export only one function `triangulate_path_edge` which 
+takes a list of points and returns a list of triangles.
+
+code works only with float32 points.
+
+```python
+from typing import Literal
+import numpy as np
+import numpy.typing as npt
+
+def triangulate_path_edge(
```

Comment:
Is this just copied over from the package? I don't know if that's a good idea (hard to keep in sync etc)

Reply:
(But, happy to merge as-is for now until we have proper autogenerated API docs)

Reply:
No. It is copied from napari code. 
I prefer to generate documentation from code but it will require time to setup